QModManager/API/ModLoading/QModPatchMethod.cs

```diff
@@ -5,6 +5,7 @@
 
     /// <summary>
     /// Identifies a normal patch method for a QMod.<para/>
+    /// This method must be public, must take no parameters, and must return either <seealso cref="void"/> or <seealso cref="PatchResults"/>.<para/>
```

Comment:
Doesnt really have to be public

Reply:
While it would be possible to make this work without it being `public`, it would be "more correct" if it were.  
This is one assembly exposing methods to be invoked by another, external assembly.  
When that is the intent, you make the method and class `public`.

Having the method be `public` is no different to how mod patching methods are currently being done.

Reply:
Doesnt it already work for non-public methods?

Reply:
I can't say for sure as I don't have any examples of a mod having these methods as non-public.  
Even if it does work without the method being public, there's no harm in asking modders to keep them public for simplicity.